source/fundamentals/builders.txt

```diff
@@ -19,35 +19,66 @@ Builders
    :depth: 2
    :class: singlecol
 
-This section includes guides on how to use each of the available builders, and demonstrates the utility the Java driver
-builder classes provide.
+This section includes guides on how to use each of the available
+builders, and demonstrates the utility the Java driver builder classes
+provide. 
 
 Overview
 --------
 
-The Java driver provides classes to make it easier to use CRUD operations and the Aggregation API. The static utility methods
-allow you to build a query more succinctly.
+The Java driver provides classes to simplify the process for developers
+to use CRUD operations and the Aggregation API. The static utility methods allow you
+to build a query more efficiently. 
 
+Why Use Builders?
+-----------------
 
-Imagine we want to send a marketing email to all users in our users collection that identify as "female" gender and are older than "29".
-We only need their email address, so we'll ensure our query doesn't return data we pay bandwidth
-costs for but don't need.
+Using the builders class, you leverage the power of:
 
-Using JSON and the MongoDB shell, the query would look something like:
+- The Java compiler and the IDE to find errors during development
+- The IDE for discovery and code completion
+
+The Java compiler and the IDE catches errors such as misspelled
+operators early on. When using the MongoDB shell or plain Java, you
+write operators as strings. You get no visual indication of a problem
+and only find it when you run the code. 
```

Comment:
```suggestion
operators early on. When using the MongoDB shell or plain Java you
write operators as strings and get no visual indication of a problem,
pushing these errors to runtime instead of compile time. 
```